assets/src/design-system/components/input/index.js

```diff
@@ -90,10 +90,14 @@ const InputContainer = styled.div(
 );
 
 const StyledInput = styled.input(
-  ({ theme }) => css`
+  ({ hasSuffix, theme }) => css`
     height: 100%;
     width: 100%;
-    padding: 0 8px 0 0;
+    padding: 0;
+    ${hasSuffix &&
+    css`
+      padding-right: 8px;
+    `}
```

Comment:
@samwhale, added this here, so there's no unnecessary padding if no suffix following the input. Just FYI.